src/classes/HH_HouseholdNaming_TEST.cls

```diff
@@ -559,4 +627,170 @@ public class HH_HouseholdNaming_TEST {
 
         System.assert(new HH_HouseholdNaming().isAutomaticNaming(), 'Automatic Household Naming should be turned on');
     }
+
+    /*********************************************************************************************************
+    @description
+        Test getHouseholdContacts() returns empty list on empty/null Account/Household Ids input arg.
+    verify:
+        An empty list is returned.
+    **********************************************************************************************************/  
+    private static testMethod void testGetHouseholdContactsReturnsEmptyList() {
+        // The private Contact should not be returned by the query
+        insert new Contact(LastName = 'foo', npe01__Private__c = true);
+        
+        HH_HouseholdNaming hhNaming = new HH_HouseholdNaming();
+
+        List<Id> hhIds = null;
+        List<Contact> contacts = hhNaming.getHouseholdContacts(hhIds);
+        System.assertEquals(0, contacts.size(), 'The Contact list should contain no record for null Id list argument: ' + contacts);
+
+        hhIds = new List<Id>();
+        contacts = hhNaming.getHouseholdContacts(hhIds);
+        System.assertEquals(0, contacts.size(), 'The Contact list should contain no record for empty Id list arg: ' + contacts);
+        
+        hhIds.add(null);
+        contacts = hhNaming.getHouseholdContacts(hhIds);
+        System.assertEquals(0, contacts.size(), 'The Contact list should contain no record when Id list arg contains null: ' + contacts);
+
+        Id mockAccountId = Account.sObjectType.getDescribe().getKeyPrefix() + '000000000001AAA';
```

Comment:
Create the mockAccountId as a constant.

Reply:
A mock Account Id has been created in `UTIL_UnitTestData_TEST` and used throughout unit tests.